tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es2018",
-    "lib": ["es2018"],
+    "target": "esnext",
```

Comment:
should target be `es2022` and lib be `es2023`? do we use next here?

Reply:
I don't know anything about these so willing to do whatever you recommend.